src/version.ts

```diff
@@ -0,0 +1,189 @@
+import chalk from 'chalk'
+import fs from 'fs'
+import FileHelpers from './filehelpers'
+// eslint-disable-next-line
+const minimist = require("minimist");
+
+const args = minimist(process.argv.slice(2))
+
+/**
+ * MiraVersion checks the CDK version consistency across the Mira packge and the
+ * applications using Mira.  It is capable of force-patching package.json files
+ * and running the NPM install.
+ */
+export class MiraVersion {
+    static instance: MiraVersion
+    cdkVersion: string
+    constructor () {
+      if (!MiraVersion.instance) {
+        MiraVersion.instance = this
+      }
+      const cdkVersion = this.getLocalMiraCDKVersion()
+      if (typeof cdkVersion === 'string') {
+        this.cdkVersion = cdkVersion
+      }
+    }
+
+    /**
+     * Checks a particular dependency CDK version.
+     */
+    /* eslint-disable-next-line */
+    checkApplicationDependencyCDKVersion (pkg: any, dep: string, autoFix = true): boolean {
+      if (!dep.startsWith('@aws-cdk') && !dep.startsWith('aws-cdk')) {
+        return false
+      }
+      const cdkVersion = pkg.dependencies[dep]
```

Comment:
The dependency can be a `devDependency` but it is always written to the `dependencies` object.

Reply:
Ah, interesting.  Will look into this and create a patch.

Reply:
This was addressed